client/my-sites/site-settings/form-general.jsx

```diff
@@ -330,6 +330,54 @@ class SiteSettingsFormGeneral extends Component {
 		);
 	}
 
+	netNeutralityOption() {
+		const { fields, isRequestingSettings, translate, handleToggle, moment, handleSubmitForm, isSavingSettings } = this.props;
+
+		const today = moment(),
+			lastDay = moment( { year: 2017, month: 6, day: 12 } );
```

Comment:
As a note, days and years are 1-indexed, and other things are 0-indexed. So, this date is 12-7-2017 for my non-US friends or 7-12-2017 for my weird US friends 😄